package.json

```diff
@@ -52,10 +52,7 @@
   },
   "pnpm": {
     "overrides": {
-      "eslint": ">=8",
-      "react-scripts>eslint-plugin-testing-library": ">=5",
-      "react-scripts>@typescript-eslint/parser": ">=5",
-      "react-scripts>@typescript-eslint/eslint-plugin": ">=5"
+      "eslint": ">=8"
```

Comment:
Still need to keep this because of `vite-plugin-eslint`, as explained in `CONTRIBUTING.md`.